app/src/main/java/woowacourse/movie/data/model/mapper/MovieMapper.kt

```diff
@@ -25,4 +26,11 @@ object MovieMapper : DomainViewMapper<Movie, MovieUiModel> {
             description = movie.description
         )
     }
+
+    fun toUi(movies: Movies): List<MovieUiModel> {
```

Comment:
```suggestion
    fun toUI(movies: Movies): List<MovieUiModel> {
```
용어를 통일해보면 어떨까요? (이하 관련 내용 동일)